gui/gui/src/TGInputDialog.cxx

```diff
@@ -151,7 +151,8 @@ Bool_t TGInputDialog::ProcessMessage(Long_t msg, Long_t parm1, Long_t)
                   case 1:
                      // here copy the string from text buffer to return variable
                      // coverity[secure_coding]
-                     strcpy(fRetStr, fTE->GetBuffer()->GetString());
+                     str = fTE->GetBuffer()->GetString();
+                     strlcpy(fRetStr, str, strlen(str)+1);
```

Comment:
This is wrong usage of `strlcpy`. As last argument, you should use size of memory allocated by `fRetStr`